### PR TITLE
Added Desktop DrinkWater Notification

### LINKED ISSUE #627: Desktop Drink Water Notification
# Aim

- To get enough water for our body.

# Details

- The desktop water notification displays the notification on the desktop every one minute or 30 seconds to the user to drink water. Because water is an essential need of our body.

### PR DESCRIPTION
**Pull request by viraldevpb**
# Description

I have fixed issue number #627 Which adds a new python script that displays a desktop drink water notification for the user and this python script runs every 30 minutes so that the user can drink enough water and stay healthy.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Desktop-drinkWater-Notification/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kaustubhgupta) [inline Desktop-drinkWater-Notification/.vscode/settings.json:1]: this file is not needed

File: Desktop-drinkWater-Notification/main.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import time
+from plyer import notification
+
+if __name__ == "__main__":
+    while True:
+        notification.notify(
+            title="Please Drink Water",
+            message="The U.S. National Academies of Sciences, Engineering, and Medicine determined that an adequate daily fluid intake is: About 15.5 cups (3.7 liters) of fluids a day for men. About 11.5 cups (2.7 liters) of fluids a day for women.",
+            app_icon = "D:\Programming Languages\Python\Projects\DrinkWater\icon.ico",
```
Comment (kaustubhgupta) [inline Desktop-drinkWater-Notification/main.py:9]:
```suggestion
            app_icon = "./Desktop-drinkWater-Notification/icon.ico",
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

File: Desktop-drinkWater-Notification/Readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+This python script helps user to drink more water in a day as per her body needed and also this python script is executed in the background and every 30 minutes it notifies user to drink water.
+
+## How to run this Script
+
+1. Clone this script Using this command `git@github.com:avinashkranjan/Amazing-Python-Scripts.git`
+2. You will Need Python to run this Progarm, we will install Python in Step 3.
+3. [Download](https://www.python.org/downloads/) Click on this link to install Python.
+4. Click on the Extensions and Install Code Runner this will help you to run this Progarm.
+5. If you want to run your Progarm in Powershell just type this command `Python main.py`
```
Comment (antrikshmisri) [inline Desktop-drinkWater-Notification/Readme.md:11]: Change this line to 
```suggestion
5. If you want to run your Progarm in Powershell just type this command `Python .\Desktop-drinkWater-Notification\main.py`
```
As , in your script the path to the icon is relative to root directory i.e. `Amazing Python Scripts`

**Review by antrikshmisri — CHANGES_REQUESTED** [review]

File: Desktop-drinkWater-Notification/Readme.md (hunk 3 above)
Comment (viraldevpb) [reply Desktop-drinkWater-Notification/Readme.md]: Resolved requested Change Check Now 👍

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

File: Desktop-drinkWater-Notification/main.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+if __name__ == "__main__":
+    while True:
+        notification.notify(
+            title="Please Drink Water",
+            message="The U.S. National Academies of Sciences, Engineering, and Medicine determined that an adequate daily fluid intake is: About 15.5 cups (3.7 liters) of fluids a day for men. About 11.5 cups (2.7 liters) of fluids a day for women.",
+            app_icon = "./Desktop-drinkWater-Notification/icon.ico",
+            timeout = 12
+        )
+        time.sleep(1800*1800)
```
Comment (santushtisharma10) [inline Desktop-drinkWater-Notification/main.py:12]:
```suggestion
        time.sleep(1800)
```

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]

File: Desktop-drinkWater-Notification/main.py (hunk 4 above)
Comment (viraldevpb) [reply Desktop-drinkWater-Notification/main.py]: Done Check Now !

**Review by santushtisharma10 — APPROVED** [review]
@viraldevpb 💯

**Review by avinashkranjan — APPROVED** [review]
LGTM.....🚀🚀🚀

**Pull request merged**